nussknacker-dist/src/universal/conf/docker-application.conf

```diff
@@ -0,0 +1,75 @@
+db {
```

Comment:
I don't think this file should be in ourt dist package

Reply:
This is configuration for standalone version of docker.